AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/participant-details/participant-details.component.html

```diff
@@ -70,7 +70,13 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-one-third contact-label">Email</div>
         <div id="participant-{{ participant?.ParticipantId }}-email" class="govuk-grid-column-two-thirds wrap">
-          {{ participant?.Email }}
+          <div *ngIf="!isJudge(participant)">{{ participant?.Email }}</div>
+          <div *ngIf="isJudge(participant) && (getJudgeEmail() !== '' || getJudgeEmail() === undefined || getJudgeEmail() !== null)">
+            {{ getJudgeEmail() }}
+          </div>
+          <div *ngIf="isJudge(participant) && (getJudgeEmail() === '' || getJudgeEmail() === undefined || getJudgeEmail() === null)">
```

Comment:
Here you can just do the opposite of above, we're checking they are falsy now, so try this:
```
isJudge(participant) || !getJudgeEmail()
```

Having said all of that, it's sometimes neater to use templates instead of mutliple ngIfs, like this:

```
<div *ngIf="isJudge(participant); else participantEmail">
  <div *ngIf="judgeEmail; else judgeNoEmail">
    {{ judgeEmail }}
  </div>
  <ng-template #judgeNoEmail>
    <div>
        TBC
    </div>
  </ng-template>
</div>

<ng-template #participantEmail>
  <div>
    {{ participant.Email }}
  </div>
</ng-template>
```

What's happening:
1. The else on the ngIf refers to a nested ng-template, which is denoted by the hash: `#judgeNoEmail`. This template will be displayed if the `ngIf` returns false.
2. I've inverted your logic to check for the judge first
3. I'm assuming the `getJudgeEmail()` is now a getter, so it becomes just `judgeEmail`
4. I've left all the divs in because I'm assuming (without checking) that they're needed for structure, i.e., block level. However, if they're being displayed inline anyway, you can remove any `<div>` and replace any divs that contain ngIfs with ng-containers (as these don't get rendered to the DOM so they keep the doc neater.